src/Component/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,36 @@
+
+import React, { useState } from 'react';
+
+export const RadioButton = () => {
+
+  const [searchType, setSearchType] = useState('organizations'); 
```

Comment:
`organizations` should be in a constant and used here

const SEARCH_TYPES = {
ORGANIZATIONS: 'organizations',
...
}